packages/js/e2e-core-tests/src/specs/merchant/wp-admin-order-edit.test.js

```diff
@@ -69,14 +77,152 @@ const runEditOrderTest = () => {
 			await utils.waitForTimeout( 2000 );
 
 			// Save the order changes
-			await expect( page ).toClick( 'button.save_order' );
-			await page.waitForSelector( '#message' );
+			await orderPageSaveChanges();
 
 			// Verify
 			await expect( page ).toMatchElement( '#message', { text: 'Order updated.' } );
-			await expect( page ).toMatchElement( 'input[name=order_date]', { value: '2018-12-14' } );
+		 	await verifyValueOfInputField( 'input[name=order_date]' , '2018-12-14' );
+		});
+
+		afterAll( async () => {
+			await withRestApi.deleteAllOrders();
```

Comment:
Hi @alopezari 👋 

We can probably delete this `afterAll` block because we already have one in [line 28](https://github.com/alopezari/woocommerce/blob/8561de2f064ba0bc60dde9c2cd144609d1dd507f/packages/js/e2e-core-tests/src/specs/merchant/wp-admin-order-edit.test.js#L28)?

Reply:
Hi @rodelgc! Yes, I missed it. Thanks for spotting this! Will fix it.

Reply:
No worries! 😄